web/src/hashchange.ts

```diff
@@ -237,6 +238,26 @@ function do_hashchange_normal(from_reload: boolean, restore_selected_id: boolean
         case "#feed":
             show_all_message_view();
             break;
+        case "#topics":
+            // Show inbox style topics list for the channel if
+            // the channel id is valid.
+            if (hash.length === 3) {
+                const channel_id_string = hash[2];
+                if (channel_id_string) {
+                    inbox_ui.show(
+                        new Filter([
+                            {
+                                operator: "channel",
+                                operand: channel_id_string,
+                            },
+                        ]),
+                    );
+                    return true;
+                }
+            }
+            // Otherwise, show home view for malformed URL.
+            show_home_view();
```

Comment:
We should track as a TODO giving an error page here.

Reply:
No longer required since I changed the code to use `#narrow` code path.